axltempl/drupal.py

```diff
@@ -256,6 +256,9 @@ def generate_drupal_files(
     os.makedirs("config/sync", mode=0o755, exist_ok=True)
     util.write_file("config/sync/.gitkeep", "")
 
+    os.makedirs(docroot + "/modules/custom", mode=0o755, exist_ok=True)
```

Comment:
@devtaher how about doing the same for `themes as well`?

Reply:
@hussainweb to maintain consistency, I think its good to have the custom themes directory also added. I have updated the PR for review.